### DISCUSSION TITLE
SvelteFlow NodeResizer can only resize accurately from bottom-right

### DISCUSSION THREAD

**Discussion by Everduin94** [xyflow/xyflow#4070] [Q&A]
Hey there. Posting this here in discussions, and not issues, as I can't reproduce in the xyflow stackblitz. - Looking for input as to what I could try / debug as I'm at a loss. 
- I'm on version `0.0.39`. 
- I have the same issue using very basic custom nodes copied from the stack blitz and an isolated node store.

**When using NodeResizer**. I can only resize (accurately) from the bottom right corner, bottom line and right line. 

**Trying to resize from any other corner or the top or left line results in inaccurate resizing, as if I was pulling the bottom or right line.**
- **Update**: When I drag from a control that should update the position. The position is not updating. Then once I drag the node, it suddenly updates it's translate value into the correct position.
  - See second video

https://github.com/xyflow/xyflow/assets/14320878/a4d4bd71-b2c6-44f4-962f-6b60b0ef69c4

https://github.com/xyflow/xyflow/assets/14320878/ea43f0c3-88bc-43b4-b7f3-27fbc862eb66

More context: I saw this issue and updated versions since all of my nodes had an `origin` of [0.5, 0] I was seeing weird behavior. https://github.com/xyflow/xyflow/pull/3963

However, even after updating (and removing the origin property) I still have the same issue.

**Comment by Everduin94** [comment]
Svelte NodeResizer updates position of the node on the store, but does not update the `transform: translate` x,y when resizing in a way that should move the node,  in Svelte version `3.59.2`. - Mentioning this because `3.59.2` still has 200k weekly downloads

https://stackblitz.com/edit/zzfqud?file=src%2Fexample%2FApp.svelte

**Comment by creationwin** [comment]
Is this a controlled-flow?
